views/forms/form_match.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * PEP Capping 2017 Algozzine's Class
+ *
+ * Matching lookup that searchs for possible duplicate participants in the db
+ *
+ * After the referral/sel-referral/ intake form is completed, the page will
+ * grab all previously entered information and attempt to find matches on this page 
+ * through an ajax request. This page will compare user input against the db and 
+ * return a list of matches that are then displayed to the user
+ *
+ * @author Vallie Joseph and Michelle Crawley
+ * @copyright 2017 Marist College
+ * @version 1.1
+ * @since 1.1
+ */
+
+authorizedPage();
+global $db, $params;
+include ("header.php");
+
+// Grab info passed from hidden for mfields
+$firstName = trim($_GET['firstname']);
+$lastName = trim($_GET['lastname']);
+
+// Query the db to find any duplicate entries
+$db->prepare("get-duplicate-data", "SELECT * FROM ParticipantModal WHERE firstname  = $1 AND lastname = $2 LIMIT 10");
+$result = $db->execute("get-duplicate-data", [$firstName, $lastName]);
+
+function checkEmpty($string){
+    if($string == ""){
+        echo "<i> N/A</i>";
+    }else{
+         echo $string;
+    }
+}
+?>
+ <ul class='list-group'>
+
+    <?php
+    // Display all matching values as a list item
+    while($row = pg_fetch_assoc($result)){
+    ?>
+         <li class="list-group-item " id="<?=$row['peopleid']?>">
```

Comment:
When hovering over these I can't tell I'm supposed to click on them. Maybe suggest `cursor: pointer`